docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -0,0 +1,101 @@
+# Configure Cloud Storage Credentials
+
+Streaming dataset supports the following cloud storage providers to stream your data directly to your instance.
+- [Amazon S3](https://aws.amazon.com/s3/)
```

Comment:
I think we should remove the links to the product pages for the cloud provider storage. If they are looking here they already know the product. It's also a bit jarring to be yanked out of the docs and pushed into the Cloud Storage pages. I do like the list upfront though.

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/169/commits/c164c1329e2a1cdfc5d49ece7647e05bca8ce411.